tcf_website/static/department/sortClasses.js

```diff
@@ -57,13 +86,56 @@ const sortByGpa = () => {
         $("#rating-sort-btn").removeClass("active");
         $("#diff-sort-btn").removeClass("active");
         $("#gpa-sort-btn").addClass("active");
+        $("#descending-order-btn").removeClass("active");
+        $("#ascending-order-btn").addClass("active");
     } else {
         sortGpaAsc *= -1;
+        if ($("#ascending-order-btn").hasClass("active")) {
+            $("#ascending-order-btn").removeClass("active");
+            $("#descending-order-btn").addClass("active");
+        } else {
+            $("#descending-order-btn").removeClass("active");
+            $("#ascending-order-btn").addClass("active");
+        }
     }
 
     sortClasses(cmpByProp("gpa", sortGpaAsc));
 };
 
+const ascendingOrder = () => {
+    if (!$("#ascending-order-btn").hasClass("active")) {
+        $("#descending-order-btn").removeClass("active");
+        $("#ascending-order-btn").addClass("active");
+        // Check which sort is currently selected and compute that sort
+        if ($("#number-sort-btn").hasClass("active")) {
+            sortClasses(cmpByProp("title", 1));
+        } else if ($("#rating-sort-btn").hasClass("active")) {
+            sortClasses(cmpByProp("rating", -1));
+        } else if ($("#diff-sort-btn").hasClass("active")) {
+            sortClasses(cmpByProp("difficulty", 1));
+        } else if ($("#gpa-sort-btn").hasClass("active")) {
+            sortClasses(cmpByProp("gpa", -1));
+        }
+    }
+};
+
+const descendingOrder = () => {
+    if (!$("#descending-order-btn").hasClass("active")) {
+        $("#ascending-order-btn").removeClass("active");
+        $("#descending-order-btn").addClass("active");
+        // Check which sort is currently selected and compute that sort
+        if ($("#number-sort-btn").hasClass("active")) {
+            sortClasses(cmpByProp("title", -1));
+        } else if ($("#rating-sort-btn").hasClass("active")) {
+            sortClasses(cmpByProp("rating", 1));
+        } else if ($("#diff-sort-btn").hasClass("active")) {
+            sortClasses(cmpByProp("difficulty", -1));
+        } else if ($("#gpa-sort-btn").hasClass("active")) {
+            sortClasses(cmpByProp("gpa", 1));
+        }
+    }
+};
```

Comment:
These two functions seem like they might be better suited in the common sorting file under static/common/sorting.js
Also, as it stands, these functions are not bound to any buttons as far as I can tell? Look under the file department/bind-sort.js for example.

Reply:
Cool, I'll definitely move those over. I think you missed it but if you scroll to the top you'll see that they are bonded in department/bind-sort.js.

Reply:
Oops my bad!